src/test/HostActivationTests/GivenThatICareAboutStandaloneAppActivation.cs

```diff
@@ -50,6 +50,12 @@ public void Running_Build_Output_Standalone_EXE_with_DepsJson_and_RuntimeConfig_
 
             var appExe = fixture.TestProject.AppExe;
 
+            // TODO: Use FS.Chmod when build utility project is converted to csproj.
```

Comment:
@eerhardt This is how I'm handling the executable permission flags problem for now. We can probably just remove this workaround when we pick up an SDK that handles this.